app/models/ccms/submission.rb

```diff
@@ -0,0 +1,140 @@
+module CCMS
+  class Submission < ApplicationRecord # rubocop:disable Metrics/ClassLength
+    include AASM
+
+    self.table_name = :ccms_submissions
```

Comment:
Having to fix the table name is often a smell. It suggest the model namespace isn't configured correctly, or the class name is poor.